### PR TITLE
Removed not used proptypes and added nametag to UI.LabelTextArea

### PR DESCRIPTION
**Pull request by KeKs0r**
I wanted to use UI.LabelTextArea with react-form-data, but that needs that all input fields have names. That possibility was not added to UI.LabelTextArea.

Alternative solution would also be to pass all attributes on UI.LabelTextArea to the textarea itself via spread.

Any preferences?

PS: I also removed not used proptypes.

### CODE REVIEW

**Comment by jossmac** [comment]
@KeKs0r we've been meaning to use spread attributes on all our form components for a while now but have been flat-out with commercial stuff.

Would you mind re-submitting your PR with this implementation? That'd be amazing

**Comment by KeKs0r** [comment]
@jossmac 
I have copied the implementation from the normal TextArea. (Including the FixMe comment).

File: lib/ui/LabelTextarea.js [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
 		var className = classnames(this.props.className, {
 			'list-item': true,
 			'field-item': true,
 			'align-top': true,
 			'is-first': this.props.first,
-			'u-selectable': this.props.disabled || this.props.readonly
+			'u-selectable': disabled
 		});
 
+		// FIXME: we can do better
```
Comment (dcousens) [inline lib/ui/LabelTextarea.js]: Oh god, its spreading

Comment (KeKs0r) [inline lib/ui/LabelTextarea.js]: Should I remove it? It thought u needed the reminder for something, since u have it in several other classes as well.

**Comment by dcousens** [comment]
Can you rebase this so there are no merge commits?

**Comment by KeKs0r** [comment]
Sorry merged also commits from my other PR into master, have removed them.

File: lib/ui/LabelTextarea.js (hunk 1 above)
Comment (dcousens) [inline lib/ui/LabelTextarea.js]: Oh, no, I just meant this way of doing things is spreading.  The `FIXME` is perfectly relevant.

File: lib/ui/LabelTextarea.js [hunk 2]
```diff
@@ -1,35 +1,39 @@
...
+		var curated = _.extend({
+			ref: this.props.inputRef
+		}, this.props);
+		delete curated.className;
+		delete curated.disabled;
+		delete curated.first;
+		delete curated.readonly;
+		delete curated.children;
+		delete curated.inputRef;
+
```
Comment (dcousens) [inline lib/ui/LabelTextarea.js]: missing `delete curated.label`

File: lib/ui/LabelTextarea.js [hunk 3]
```diff
@@ -1,35 +1,39 @@
...
 			'field-item': true,
 			'align-top': true,
 			'is-first': this.props.first,
-			'u-selectable': this.props.disabled || this.props.readonly
+			'u-selectable': disabled
 		});
 
+		// FIXME: we can do better
+		var curated = _.extend({
+			ref: this.props.inputRef
```
Comment (dcousens) [inline lib/ui/LabelTextarea.js]: `this.props.inputRef` isn't used anywhere, remove it :)

File: lib/ui/LabelTextarea.js [hunk 4]
```diff
@@ -1,35 +1,39 @@
...
+		}, this.props);
+		delete curated.className;
+		delete curated.disabled;
+		delete curated.first;
+		delete curated.readonly;
+		delete curated.children;
+		delete curated.inputRef;
```
Comment (dcousens) [inline lib/ui/LabelTextarea.js]: again, `inputRef` doesn't exist, remove :)

**Comment by dcousens** [comment]
Good to merge after the remaining fixes are added :)

Thanks for following up with this @KeKs0r

**Comment by dcousens** [comment]
@JedWatson good to merge

**Comment by JedWatson** [comment]
:+1:

**Pull request merged**